cdap-tms/src/main/java/co/cask/cdap/messaging/store/leveldb/LevelDBMessageTable.java

```diff
@@ -149,7 +136,7 @@ public void close() throws IOException {
     resultSize += (payload == null) ? 0 : payload.length;
     byte[] result = new byte[resultSize];
     result[0] = 1;
-    Bytes.putLong(result, 1, txWritePtr);
+    Bytes.putBytes(result, 1, txWritePtr, 0, txWritePtr.length);
     if (payload != null) {
       Bytes.putBytes(result, 1 + Long.BYTES, payload, 0, payload.length);
```

Comment:
`Long.BYTES` is only in Java 8. Java 7 doesn't have it. Are you configuring your IntelliJ project with Java8??